cmd/crc/cmd/root.go

```diff
@@ -41,6 +41,9 @@ var (
 )
 
 func init() {
+	// Setting up logrus
+	logging.InitLogrus(logging.LogLevel, constants.LogFilePath)
+
```

Comment:
This breaks `crc xxx --log-level debug`, we'd need a `logging.SetLevel(logging.LogLevel)` call in `runPrerun()` (and to implement that method first :)

Reply:
oh thanks! I will look at this.

Reply:
I give up on this for now.